src/main.rs

```diff
@@ -178,22 +154,24 @@ fn command_new_peer(cfg: &mut configs::WireguardNetworkInfo, matches: &clap::Arg
         ips: vec![],
     };
 
-    if matches.is_present("astemplate") {
-        peer.flags.insert(0, configs::PeerFlag::Template);
+    if let Some(template_name) = matches.value_of("use-template") {
+        if let Some(template) = cfg.by_name(template_name) {
+            peer.flags = template.flags.clone();
```

Comment:
This way if we create new peer based on a template, it also becomes a template (template flags is copied).

Reply:
good find)